src/components/curriculum/CurriculumReactBootcamp.js

```diff
@@ -57,17 +57,20 @@ const CurriculumReactBootcamp = ({
       <H4>Course outline:</H4>
       <Section
         {...commonProps}
-        title="Evening pre-bootcamp (optional)"
-        name="day0"
-        subTitle={`React 101 and JS fundamentals (${trainingDateByDay({
+        title={`Evening pre-bootcamp (optional) ${trainingDateByDay({
           training,
-        })}  ${meetupTime})`}
+        })}  ${meetupTime}`}
+        name="day0"
+        subTitle={`React 101 and JS fundamentals`}
       >
         <ReactJS101Session />
       </Section>
       <Section
         {...commonProps}
-        title="React Bootcamp Day 1"
+        title={`Day 1 (${trainingDateByDay({
+          training,
+          day: 1,
+        })} ${trainingTime})`}
```

Comment:
@ric9176. What I'm saying here is not from this PR, but I just noticed it on line 50 looking at this file. I think trainingTime should be read from the training (so API) if there is a training passed to the curriculum component. So this way by default the time in the landing is the standard from 9 to 6:30 am, but in the instance, it can be different if we decide to run it in a city different timing for whatever reason. Example Berlin from 8:30 to 6pm cause german like to start working very early :-P

Reply:
@ric9176 we have the same issue with the meetupTime, which might be more likely to change based on the city. Unfortunately, this info is not actually coming from the API directly linked to the training instance, so we can't get the meetupTime from the API at this point. I think we need to soon rethink how meetups work in UM. It'd be nice to link a meetup with a training instance, so we could display the time from the API, and even link the meetup page from the training instance page if we wanted.